recipes/lmfit/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.8.3" %}
+
+package:
+  name: lmfit
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/lmfit/lmfit-py.git
+  git_tag : {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
does this use numpy's C api?  If so, you need to add x.x here and in the run req's to pin the version.